src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpSendToInteractive.cs

```diff
@@ -211,6 +213,7 @@ public void AddAssemblyReferenceAndTypesToInteractive()
             this.SubmitText("public class MyClass { public string MyFunc() { return \"MyClass.MyFunc()\"; } }");
             this.SubmitText("(new MyClass()).MyFunc()");
             this.WaitForLastReplOutput("\"MyClass.MyFunc()\"");
+            this.WaitForAsyncOperations(FeatureAttribute.SolutionCrawler);
```

Comment:
@ivanbasov Why was it necessary to have a wait at the end of the test? That seems surprising to me.